.github/workflows/test-coverage.yml

```diff
@@ -0,0 +1,46 @@
+name: Post-Merge Coverage Update
+# This should be installed in any repo where you wish to automatically update code coverage.
+# This assumes you have configured posttest to run jest-it-up and test:coverage configured to run
+# jest with coverage. This will run on all pushes to the dev branch and assumes the unit tests will pass.
+on:
+  push:
+    branches:
+      - dev
+env:
+  NODE_VERSION: ${{ vars.NODE_VERSION }}
+  RUST_VERSION: ${{ vars.RUST_VERSION }}
+jobs:
+  update-jest-it-up:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: dev
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ env.RUST_VERSION }}
+          override: true
+
+      - name: Install dependencies and check coverage
+        run: |
+          npm ci
+          npm run test:coverage
+          npm run posttest
+
+      - name: Check for config changes
+        id: check_for_changes
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+          git diff --quiet || git commit -am "chore: Update coverage thresholds"
```

Comment:
should only git add the jest config to avoid accidently updating package-lock

Reply:
Good idea we should lock this down only the `jest.config.js`.
we should be running `npm ci` instead of `npm install` in these workflows so package-lock _should not_ be getting updated by these builds.

Reply:
resolving. we decided the devs should commit coverage updates, not the workflow